components/editor/modules/figure/caption.js

```diff
@@ -35,16 +37,31 @@ const getSerializer = options => {
     parent,
     rest
   ) => {
-    return {
+    const captionNodes = node.children.filter(n => n.type !== 'emphasis')
+    const byline = node.children.find(n => n.type === 'emphasis')
+    const bylineNodes = byline.children || []
+
+    const res = {
       kind: 'block',
       type: options.TYPE,
-      nodes: inlineSerializer.fromMdast(
-        node.children,
-        0,
-        node,
-        rest
-      )
+      nodes: [
+        {
+          kind: 'block',
+          type: 'CAPTION_TEXT',
+          nodes: inlineSerializer.fromMdast(
```

Comment:
please pass along all `fromMdast` arguments. Missing: index, parent and rest.

Same below